main.go

```diff
@@ -105,12 +108,6 @@ func main() {
 		os.Exit(0)
 	}
 
-	watchNamespace, err := getWatchNamespace()
```

Comment:
i don't think we need to change any code in the main.go, besides activating the new controller